pingen/ir_attachment.py

```diff
@@ -92,12 +92,14 @@ def _handle_pingen_document(self, cr, uid, attachment_id, context=None):
                 document.write({'state': 'canceled'}, context=context)
         return
 
+    @api.model
     def create(self, cr, uid, vals, context=None):
         attachment_id = super(ir_attachment, self).create(cr, uid, vals, context=context)
         if 'send_to_pingen' in vals:
             self._handle_pingen_document(cr, uid, attachment_id, context=context)
         return attachment_id
 
+    @api.model
```

Comment:
Write shouldn't be `@api.model`.

Reply:
Moreover, if you are adding the decorator, you better change the function signature.